addon-test-support/interactor.js

```diff
@@ -0,0 +1,60 @@
+import { triggerEvent } from '@ember/test-helpers';
+import $ from 'jquery';
+
+const MONTH_SELECTOR = '.pika-lendar:visible .pika-select-month';
+const YEAR_SELECTOR = '.pika-lendar:visible .pika-select-year';
+
+const PikadayInteractor = {
+  /**
+   * @param {Date} date
+   */
+  async selectDate(date) {
+    const day = date.getDate();
+    const month = date.getMonth();
+    const year = date.getFullYear();
+    const selectEvent = 'ontouchend' in document ? 'touchend' : 'mousedown';
+
+    $(YEAR_SELECTOR).val(year);
+    await triggerEvent($(YEAR_SELECTOR)[0], 'change');
```

Comment:
It would be nice to remove the remaining usage of jQuery in here, but some of these selectors will be pretty gross to construct without it. Not sure if it's worth the effort, but dropping the jQuery dependency would be nice.